beheer.md

```diff
@@ -104,13 +104,13 @@ docker service ps getstartedlab_web | Haal de gegevens op van alle tasks die de
 docker stack rm getstartedlab | Afsluiten van de applicatie.
 docker swarm leave --force | Afsluiten van de swarm.
 
-**Criteria**
-
-* API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn
-* Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
-* De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
-* Het gebruikte Informatiemodel (minimaal UML)
-* Geautomatiseerde test en buildstraat (CI) moet beschikbaar voor de RI
-* De standaard moet ingebouwd zijn bij leveranciers (3 of meerdere leveranciers)
-* De standaard bevat een tabel waarin wordt aangegeven welke API versies met welke andere API versies compatible zijn.
+##Criteria##
+
+- API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn. Dit betekent dat zowel de Semantische Informatiemodellen (SIM), Uitwisselings Gegevensmodellen (UGM) als het Bericht Specificatiemodel (BSM) in UML beschikbaar moeten zijn.
+- Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
+- De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
+- Het gebruikte Informatiemodel (minimaal UML)
+- Geautomatiseerde test en buildstraat (CI) moet beschikbaar voor de RI
+- De standaard moet ingebouwd zijn bij leveranciers (minimaal 1 provider applicatie en 3 of meerdere consumer applicaties). 
```

Comment:
Klinkt als een kip-ei probleem? Als er nog geen standaard is, kunnen leveranciers hem ook niet gebruiken?

Reply:
Het idee is dat een standaard in de praktijk beproefd is bij tenminste één gemeente. Dan is het niet alleen een theoretische standaard maar zijn de eerste kinderziekten er uit. Dit ook om te voorkomen dat net als vroeger een mooie standaard bedacht wordt die in de praktijk niet voldoet. Bij het vaststellen van een standaard zoals Zaak- Documentservices werden geen gebruikerservaringen meegenomen en pas een tijdje later kwamen de reacties. Dat is natuurlijk veel te laat, dat moet voordat een standaard een standaard wordt.

Nu ik er zo over nadenk is het dan nog geen standaard maar een proposal denk ik?

Reply:
Als een standaard in deze situatie door een leverancier wordt ingebouwd kan het denk ik i.d.d. nog geen goedgekeurde standaard zijn. Nee het feit dat de standaard al ergens succesvol is ingebouwd is een voorwaarde om hem goed te keuren. Indien er dus in de proposal versie nog foutjes ontdekt worden moet de nieuwe proposal versie waarin de foutjes gecorrigeerd zijn weer opnieuw bij de leverancier ingebouwd worden. Dat betekent uiteindelijk dat zodra de standaard wordt goedgekeurd de betreffende leverancier meteen een goedgekeurde standaard heeft ingebouwd.

Reply:
Ik begrijp de wens wel maar dit betekend in de praktijk dat je altijd moet samenwerken met een leverancier die de API, al **voordat** het een standaard is en **voordat** deze officieel in beheer is, gaat inbouwen.

Klinkt alsof je dan altijd 1 leverancier voortrekt wat mij onwenselijk lijkt.

Ik zou dit niet als voorwaarde gaan stellen.

Reply:
Hmmm, ik begrijp wat je bedoelt. Dat is echter niet mijn bedoeling.

Ik ga er vanuit dat bij aanvang van de standaard al duidelijk is dat er daadwerkelijk behoefte aan is en dat we er vanuit mogen gaan dat de standaarden na gereed komen ook al snel door een of meerdere leveranciers ingebouwd zullen worden. In dat geval zouden we er voor kunnen kiezen om met de overdracht naar beheer te wachten totdat de eerste leveranciers deze hebben ingebouwd.

Reply:
Ik wil graag nog een nuancering aanbrengen. Het gaat er niet zozeer om dat leveranciers de standaard hebben ingebouwd maar juist dat de standaard bij enkele gemeenten is geïmplementeerd. Waarmee wordt aangetoond dat de standaard werkt. Voor de ZGW standaarden wordt dit geborgd doordat enkele bij het project betrokken gemeenten de standaard al aan het inbouwen zijn.

Dit schrijvende realiseer ik me dat dit niet zozeer een acceptatie criterium voor het in beheer nemen van een standaard maar juist een criterium voor het goedkeuren van de standaard. 

Ik stel dus voor dat we deze eis uit beheer.md halen en inbrengen in een document waarin de criteria staan beschreven waaraan een API standaard moet voldoen om goedgekeurd te worden.
**Let op!** Ik bedoel dus niet een document waarin staat waaraan de ZGW API standaard moet voldoen om goedgekeurd te worden.

Reply:
> Dit schrijvende realiseer ik me dat dit niet zozeer een acceptatie criterium voor het in beheer nemen van een standaard maar juist een criterium voor het goedkeuren van de standaard.

Dat klinkt prima.

Ik realiseer me net dat ZDS 1.2 met deze of de vorige criteria versie, ook geen standaard kan zijn ;-)

> Ik stel dus voor dat we deze eis uit beheer.md halen

Akkoord.

> en inbrengen in een document waarin de criteria staan beschreven waaraan een API standaard moet voldoen om goedgekeurd te worden.
> Let op! Ik bedoel dus niet een document waarin staat waaraan de ZGW API standaard moet voldoen om goedgekeurd te worden.

Huh, waarom moeten de ZGW APIs er niet aan voldoen?

Reply:
> Dit schrijvende realiseer ik me dat dit niet zozeer een acceptatie criterium voor het in beheer nemen van een standaard maar juist een criterium voor het goedkeuren van de standaard.

Dat klinkt prima.

Ik realiseer me net dat ZDS 1.2 met deze of de vorige criteria versie, ook geen standaard kan zijn ;-)

> Ik stel dus voor dat we deze eis uit beheer.md halen

Akkoord.

> en inbrengen in een document waarin de criteria staan beschreven waaraan een API standaard moet voldoen om goedgekeurd te worden.
> Let op! Ik bedoel dus niet een document waarin staat waaraan de ZGW API standaard moet voldoen om goedgekeurd te worden.

Huh, waarom moeten de ZGW APIs er niet aan voldoen?

Reply:
Ik denk dat Robert bedoelt:
De ZGW API's moeten er wel aan voldoen maar het is geen document wat exclusief voor de ZGW API's geschreven is, maar algemeen geldend is voor alle API standaarden die we in beheer gaan nemen.

Reply:
> Ik realiseer me net dat ZDS 1.2 met deze of de vorige criteria versie, ook geen standaard kan zijn ;-)

Volgens de oude spelregels nog wel :-D Maar je hebt gelijk, de oude Zaak- Documentservices (en andere StUF koppelvlakken) zijn pas in de praktijk beproefd nadat de standaard vastgesteld was. Dus bij de eerste implementaties zijn de kinderziekten naar boven gekomen die er eigenlijk bij de openbare consultatie (toen nog een papieren traject) uit hadden moeten komen.

Reply:
De eis staat er op dit moment nog wel in maar anders verwoord.
Pas als er een document is waarin staat waar API standaarden aan moeten voldoen en deze eis daarin is opgenomen kan hij uit deze criteria worden verwijderd.